avm/src/vm/event.rs

```diff
@@ -213,9 +213,11 @@ impl HandlerFragment {
           for i in 0..instructions.len() {
             let ins = &instructions[i];
             let async_func = ins.opcode.async_func.unwrap();
+            //eprintln!("{} {} {} {}", ins.opcode._name, ins.args[0], ins.args[1], ins.args[2]);
```

Comment:
leaving these here since we have similar ones in line 231 and they were very useful. Also ok removing all of them, but prefer to just keep them around